src/gui/guiHyperText.h

```diff
@@ -188,7 +188,8 @@ class TextDrawer
 	};
 
 	ParsedText m_text;
-	Client *m_client;
+	Client *m_client; //< may be null
+	ISimpleTextureSource *m_tsrc;
 	gui::IGUIEnvironment *m_environment;
```

Comment:
as an aside it would be good to rename this to `m_guienv` for consistency someday.